ynr/apps/elections/templates/elections/ballot_view.html

```diff
@@ -241,9 +241,24 @@ <h4>Add a new candidate</h4>
   {% if ballot.has_results and ballot.resultset %}
     {% include "elections/includes/_ballot_results_table.html" with results=ballot.resultset %}
   {% endif %}
+
+  {% if user.is_authenticated %}
+    <h2>History for this ballot</h2>
+    {% for action in logged_actions %}
```

Comment:
We have a pattern for recent changes on https://candidates.democracyclub.org.uk/recent-changes. It would be nice if every "log of changes" on the site looked more or less the same. Can you convert this to a table with the same column headings, where possible? I get we can't do a diff, but we should be able to show most of the other columns